src/exampleTypes/Bool.js

```diff
@@ -1,18 +1,23 @@
 import React from 'react'
+import _ from 'lodash/fp'
 import F from 'futil-js'
-import { contexturify } from '../utils/hoc'
-import RadioListDefault from '../layout/RadioList'
+import { contexturify, defaultTheme } from '../utils/hoc'
+import RadioList from '../layout/RadioList'
```

Comment:
One of the nice things about the `defaultTheme` pattern is not having to worry about shadowing component imports in prop names anymore